lms/djangoapps/certificates/admin.py

```diff
@@ -57,9 +58,18 @@ class CertificateTemplateAssetAdmin(admin.ModelAdmin):
     """
     Django admin customizations for CertificateTemplateAsset model
     """
+
     list_display = ('description', 'asset_slug',)
     prepopulated_fields = {"asset_slug": ("description",)}
 
+    # see PROD-1153 for the details
+    def changelist_view(self, request, extra_context=None):
+        if '.stage' in request.get_host():
```

Comment:
Is this change valid for openedx?

Reply:
django messages are usually used for flashed messaging. 
Regarding openedx you are right. I think we can add flag for that.